platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ExportPathMigrationException.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.migration;
+
+import java.nio.file.Path;
+
+/**
+ * Provides error information related to an exported path that was not included in the exported file.
+ * <p>
+ * <b>
+ * This code is experimental. While this interface is functional
+ * and tested, it may change or be removed in a future version of the
+ * library.
+ * </b>
+ * </p>
+ */
+public class ExportPathMigrationException extends ExportMigrationException {
+    private static final String ABSOLUTE_PATH_WARNING = "Path [%s] %s";
+
+    private static final String SYSTEM_PROPERTY_ABSOLUTE_PATH_WARNING =
+            "The value for system property [%s] is set to a path [%s] that %s";
+
+    private static final String JAVA_FILE_PROPERTY_ABSOLUTE_PATH_WARNING =
+            "The value for property [%s] in file [%s] is set to a path [%s] that %s";
+
+    /**
+     * Constructor used to report a migration error on a specific path.
+     *
+     * @param path  path that cannot be exported
```

Comment:
😉  Not my original code. I simply moved it there. I will update all of that as part of the exception re-write mentioned above.